extensions/EXT/EGL_EXT_gl_colorspace_display_p3_passthrough.txt

```diff
@@ -0,0 +1,142 @@
+Name
+
+    EXT_gl_colorspace_display_p3_passthrough
+
+Name Strings
+
+    EGL_EXT_gl_colorspace_display_p3_passthrough
+
+Contributors
+
+    Chris Forbes
+    Courtney Goeltzenleuchter
+
+Contact
+
+    Courtney Goeltzenleuchter (courtneygo 'at' google.com)
+
+IP Status
+
+    No known IP claims.
+
+Status
+
+    Draft
+
+Version
+
+     Version 1 - Dec 4, 2018
+
+Number
+
+    EGL Extension #130
+
+Extension Type
+
+    EGL display extension
+
+Dependencies
+
+    This extension is written against the wording of the EGL 1.5
+    specification (August 27, 2014).
+
+    This extension requires EGL_KHR_gl_colorspace.
+
+Overview
+
+    Applications that want to use the Display-P3 color space (DCI-P3 primaries
+    with sRGB-like transfer function) can use this extension to
+    communicate to the platform that framebuffer contents represent colors in
+    the non-linear Display-P3 color space.
+    The application is responsible for producing appropriate framebuffer
+    contents. An application would want to use this extension rather than
+    EGL_EXT_gl_colorspace_display_p3 if they apply the sRGB transfer function
+    themselves and do not need the HW to do it.
+
+New Procedures and Functions
+
+    None.
+
+New Tokens
+
+    Accepted as attribute values for EGL_GL_COLORSPACE by
+    eglCreateWindowSurface, eglCreatePbufferSurface and eglCreatePixmapSurface:
+
+    [[ If EGL_EXT_gl_colorspace_display_p3_linear is supported ]]
+
+        EGL_GL_COLORSPACE_DISPLAY_P3_PASSTHROUGH_EXT         0x3490
+
+Modifications to the EGL 1.5 Specification
+
+    Insert below text in the 3rd paragraph on page 33 in 3.5.1 "Creating On-
+    Screen Rendering Surfaces, before "The default value of EGL_GL_COLORSPACE
+    is EGL_GL_COLORSPACE_LINEAR.":
+
+    [[ If EGL_EXT_gl_colorspace_display_p3_passthrough is supported ]]
+
+    If its value is EGL_GL_COLORSPACE_DISPLAY_P3_PASSTHROUGH_EXT, then a
+    non-linear, sRGB encoded Display-P3 color space is assumed, with a
+    corresponding GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING value of GL_LINEAR.
+    The application is responsible for applying the appropriate transfer
+    function when writing and reading pixels.
+
+    Modify the 4th paragraph on the same page:
```

Comment:
"Insert below text after the 4th paragraph on the same page"

Reply:
Fixed.